recenter_tc/plugins/modules/output_checkers/fdeck_checker.py

```diff
@@ -70,20 +75,26 @@ def test_product_recenter_tc(
     return goodcomps, badcomps, compare_strings
 
 
-def compare_outputs_recenter_tc(compare_path, output_products):
-    """Compare the "correct" imagery found in comparepath with the list of current output_products
+def call(plugin, compare_path, output_products, test_product_func=None):
+    """Compare the "correct" imagery found in comparepath with list of output_products.
 
-    Args:
-        comparepath (str) : Path to directory of "correct" products - filenames must match output_products
-        output_products (list) : List of strings of current output products, to compare with products in compare_path
+    Parameters
+    ----------
+    plugin: OutputCheckerPlugin
+        - The corresponding fdeck OutputCheckerPlugin that has access to needed methods
+    comparepath (str) :
+        - Path to directory of "correct" products - filenames must match
+          output_products
+    output_products (list) :
+        - List of strings of current output products, to compare with products in
+          compare_path
 
-    Returns:
-        int: Binary code: Good products, bad products, missing products
+    Returns
+    -------
+    int: Binary code: Good products, bad products, missing products
     """
-
     LOG.info("Using compare_outputs_recenter_tc")
-    retval = compare_outputs(
+    retval = plugin.compare_outputs(
         compare_path, output_products, test_product_func=test_product_recenter_tc
     )
```

Comment:
Do we think it is ok for all of the "call" methods on all of the output checkers to basically be identical?  I think they might be exactly identical (if we remove the "test_product_func" kwarg).  Is there some other way this should be organized so we can consolidate the identical "call" methods (though perhaps it is just that "you can compare outputs however you want, but if you want to use this built in "compare_outputs" utility, you can. In which case this is probably the way to do it).

Reply:
That was pretty simplified! Resolving!